src/libs/ajax/data-table-providers/WdsDataTableProvider.test.ts

```diff
@@ -17,12 +17,16 @@ const uuid = '123e4567-e89b-12d3-a456-426614174000' // value doesn't matter for
 
 // shell class that extends WdsDataTableProvider to allow testing protected methods
 class TestableWdsProvider extends WdsDataTableProvider {
-  transformPageOverride(wdsPage: RecordQueryResponse, recordType: string, queryOptions: EntityQueryOptions): EntityQueryResponse {
-    return this.transformPage(wdsPage, recordType, queryOptions)
+  transformPageOverride(wdsPage: RecordQueryResponse, recordType: string, queryOptions: EntityQueryOptions, metadata: EntityMetadata): EntityQueryResponse {
+    return this.transformPage(wdsPage, recordType, queryOptions, metadata)
+  }
+
+  transformAttributesOverride = (attributes: RecordAttributes, primaryKey: string): RecordAttributes => {
```

Comment:
Just a question for curiosity here -- why the decision to override the function here? Rather than just calling it as is?

Reply:
the original method - `transformPage` - is protected, and thus can't be called from anywhere except within its own class or from an extending class. So I made `TestableWdsProvider` as an extending class.